src/DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -12,11 +12,16 @@
 use Doctrine\Bundle\MongoDBBundle\Fixture\ODMFixtureInterface;
 use Doctrine\Bundle\MongoDBBundle\Mapping\Driver\XmlDriver;
 use Doctrine\Bundle\MongoDBBundle\Repository\ServiceDocumentRepositoryInterface;
+use Doctrine\Bundle\MongoDBBundle\Tests\Fixtures\Validator\EmbeddedDocument;
```

Comment:
I believe you imported the wrong class - this should be `Doctrine\ODM\MongoDB\Mapping\Annotations\EmbeddedDocument`